tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "compilerOptions": {
-    "target": "ES2017",
+    "target": "es2018",
```

Comment:
Should this match below?

Reply:
Yeah, I don't think it really matters with Next.js since it will be bundling it. The new next starters use `esnext` for both, so I'll just do that.

Reply:
Updated to conform more to the current Next style. They also use `bundler` as the resolver.